src/topology/separation.lean

```diff
@@ -16,6 +16,61 @@ variables {α : Type u} {β : Type v} [topological_space α]
 
 section separation
 
+/--
+`separate` is a predicate on pairs of `finset`s of a topological space.  It holds if the two
+`finset`s are contained in disjoint open sets.
+-/
+def separate : finset α → finset α → Prop :=
+  λ (s t : finset α), disjoint s t → ∃ U V : (set α), (is_open U) ∧ is_open V ∧
```

Comment:
I'm confused... do you really want `disjoint s t ->`, or do you want `disjoint s t \and`?